doc/source/Contributing.rst

```diff
@@ -16,4 +16,32 @@ Adding a New Parameter
 
   2. Update the fortran definition for the ``grackle_chemistry_data`` type (in ``src/clib/grackle_fortran_interface.def``). This type must exactly match the definition of :c:data:`chemistry_data` and is used to integrate Grackle into simulation codes written in Fortran.
 
-  3. Add docummentation in ``doc/source/Parameters.rst`` for your new parameter.
+  3. Add documentation in ``doc/source/Parameters.rst`` for your new parameter.
+
+Instructions for Making a Release
+---------------------------------
+
+Before reading this section, please ensure you are familiar with our :ref:`versioning policy <versioning-code>`.
+
+To create a new release:
+
+  1. Draft the changes to the changelog and draft the release notes (GitHub allows you to circulate a draft of the release notes with other developers).
```

Comment:
It may worth mentioning here that it's easier to start with the github release draft as you can have it automatically list all the PRs since the last relase.

Reply:
Great Point! I didn't actually know that